lib/Doctrine/ORM/Query/Expr.php

```diff
@@ -526,13 +526,13 @@ public function notLike($x, $y)
      * Creates a CONCAT() function expression with the given arguments.
      *
      * @param mixed $x First argument to be used in CONCAT() function.
-     * @param mixed $y Second argument to be used in CONCAT() function.
+     * @param mixed $y,... Other arguments to be used in CONCAT() function.
      *
      * @return Expr\Func
      */
     public function concat($x, $y)
```

Comment:
@zeroedin-bill I agree with you. I was just suggesting that the scenario is rare and not recommended due to the Builder pattern, so the maintainer may decide to merge it anyway. Also, this can be merged in master, only partially back-ported to 2.4 and/or 2.5.